actions/universal-naming-conventions.js

```diff
@@ -1,14 +1,22 @@
 module.exports = (course, item, callback) => {
```

Comment:
This shouldn't contain any changes in other files than the `set-external-links` grandchild. Can we make sure these changes don't affect anything else?